client/src/components/chess-square/ChessSquare.test.js

```diff
@@ -17,13 +17,13 @@ describe('<ChessSquare />', () => {
   });
 
   test('should render chess square with piece', () => {
-    const wrapper = shallow(<ChessSquare isDark pieceType="b" pieceColor="b" gridPosition="c8" highlight={false} />);
+    const wrapper = shallow(<ChessSquare isDark pieceType="p" pieceColor="b" gridPosition="c8" highlight={false} handleSelect={fn} />);
```

Comment:
Will we need a constant file to represent these values (p, b, etc). We may be able to reduce uninted code errors that way.

Reply:
LGTM after you finalise on the constants. If not required, don't bother spending time there.